model/config.go

```diff
@@ -312,9 +312,11 @@ func (s *ServiceSettings) SetDefaults(isUpdate bool) {
 	if s.EnableEmailInvitations == nil {
 		// If the site URL is also not present then assume this is a clean install
 		if s.SiteURL == nil {
-			s.EnableEmailInvitations = NewBool(false)
-		} else {
-			s.EnableEmailInvitations = NewBool(true)
+			if s.EnableDeveloper != nil && *s.EnableDeveloper {
```

Comment:
@reflog, I think you meant to change the block below on line 323. This has removed the defaulting of `s.EnableEmailInvitations` altogether, which probably explains the failed build.

Reply:
Oh my, how many typos can one man make! :)
Sorry, fixed now, let's see if the build survives.